src/contractTest/java/uk/gov/hmcts/reform/sscs/ContractTestDataProvider.java

```diff
@@ -142,12 +142,12 @@ protected static HearingDetails hearingDetails() {
         hearingDetails.setDuration(1);
         hearingDetails.setNonStandardHearingDurationReasons(Arrays.asList("First reason", "Second reason"));
         hearingDetails.setHearingPriorityType("Priority type");
-        HearingLocations location1 = new HearingLocations();
+        HearingLocation location1 = new HearingLocation();
         location1.setLocationId("court");
         location1.setLocationType("Location type");
-        List<HearingLocations> hearingLocations = new ArrayList<>();
-        hearingLocations.add(location1);
-        hearingDetails.setHearingLocations(hearingLocations);
+        List<HearingLocation> hearingLocation = new ArrayList<>();
```

Comment:
a list is generally not singular

Reply:
this will have multiple in some cases in a upcoming ticket.

Reply:
That would make it immutable

Reply:
As discussed, this is because further locations can be added later.  Agreed Lucas, nice work ;)